Rakefile

```diff
@@ -1,6 +1,16 @@
-require 'bundler/gem_tasks'
-require 'rspec/core/rake_task'
+# frozen_string_literal: true
+
+require "bundler/gem_tasks"
```

Comment:
Oh, this fight " vs ' 😄 

I'm fine with any if it's controlled by a lint and consistent in the project 👍

Reply:
Oh yes 😄 , I thought for a very long time what is better to use. As a result, I decided to use what `Standard` offers (`"` - default behavior for `Standard` and its controlled by him)